features/test_runner/fastlane_test_runner.rb

```diff
@@ -1,24 +1,29 @@
 require_relative "./fastlane_test_runner_helpers/fastlane_ci_output"
+require_relative "test_runner"
 
 module FastlaneCI
   # Represents the test runner responsible for loading and running
   # fastlane Fastfile configurations
+  # TODO: run method *should* return an array of artifacts
   class FastlaneTestRunner < TestRunner
     def run(platform: nil, lane: nil, parameters: nil)
       require "fastlane"
 
       ci_output = FastlaneCI::FastlaneCIOutput.new(
         file_path: "fastlane.log",
-        block: proc do |row|
+        each_line_block: proc do |row|
           puts "Current output from fastlane: #{row}"
-          yield(row)
+          # WARN: This crashes because (not block provided)
+          #yield(row)
         end
       )
       FastlaneCore::UI.ui_object = ci_output
 
       # this only takes a few ms the first time being called
       Fastlane.load_actions
 
+      # TODO: How do I access fastlane actions (like lane_context) from here?
```

Comment:
I'm kind of stuck here.

Reply:
Alright, just took a look, and @taquitos is gonna love this. The `lane_context` isn't attached to the fastlane runner, but instead, you can access it right after the `fast_file.runner.execute(self.lane, self.platform, self.parameters)` call using

```ruby
Actions.lane_context
```
this prints out something like
```ruby
=> {:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios beta"}
```

as mentioned before though, for now, please just assume some dummy/test values, as I'm currently working on this file with my current PR of https://github.com/fastlane/ci/pull/200

Reply:
That's great! I'll try to get some work on that, is the last piece of the puzzle, maybe it'd be great to wait to your #200 to finish stablishing how the `FastlaneRunner` will work. What do you think?